src/javascript/app/plugins/forecolor.ngWig.js

```diff
@@ -3,10 +3,17 @@ angular.module('ngWig')
     ngWigToolbarProvider.addCustomButton('forecolor', 'nw-forecolor-button');
   }])
   .component('nwForecolorButton', {
-    template: '<button colorpicker ng-model="fontcolor" ng-disabled="editMode" colorpicker-position="right" class="nw-button font-color" title="Font Color" ng-disabled="isDisabled">Font Color</button>',
+    bindings: {
+      execCommand: '=',
+      editMode: '=',
+      disabled: '='
+    },
+    template: '<button colorpicker ng-model="fontcolor" ng-disabled="$ctrl.editMode || $ctrl.disabled" colorpicker-position="right" class="nw-button font-color" title="Font Color">Font Color</button>',
     controller: function($scope) {
+      var ctrl = this;
+      
       $scope.$on('colorpicker-selected', function($event, color) {
```

Comment:
also because of babel you can use arrow functions here:

    $scope.$on('colorpicker-selected', ($event, color) => {

without losing "this" context